<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera</class>
 <widget class="QWidget" name="camera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能家居-使徒行者</string>
  </property>
  <widget class="QLabel" name="DisplayLabel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>541</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QLabel" name="Titlelabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>智能家居视频监控室</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatusLabel">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="TimeLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>60</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <property name="title">
    <string>截图显示</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>541</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 255, 0);</string>
   </property>
   <property name="title">
    <string>实时视频监控</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="CapturePushButton">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>截图</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="SavePushButton">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="ExitPushButton">
      <property name="font">
       <font>
        <family>华文楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
